f2/src/forms/ContactInfoForm.js

```diff
@@ -123,22 +123,55 @@ export const ContactInfoForm = (props) => {
                           id="email"
                         />
                         {errors.phone && (
-                          <P
-                            color="#dc3545"
-                            fontSize="1.25rem"
-                            marginBottom="0.5rem"
-                          >
-                            {phone.errorMessage}
-                          </P>
+                          <Container>
+                            <FormRow id="PhoneError">
+                              <P
+                                color="#dc3545"
+                                fontSize="1.25rem"
+                                marginBottom="0.5rem"
+                              >
+                                {phone.errorMessage}
+                              </P>
+                            </FormRow>
+                          </Container>
                         )}
-                        <Field
-                          name="phone"
-                          label={<Trans id="contactinfoPage.phoneNumber" />}
-                          component={Input}
-                          onBlur={handleBlur}
-                          onChange={handleChange}
-                          id="phone"
-                        />
+                        {errors.extension && (
+                          <Container>
+                            <FormRow id="ExtensionError">
+                              <P
+                                color="#dc3545"
+                                fontSize="1.25rem"
+                                marginBottom="0.5rem"
+                              >
+                                {extension.errorMessage}
+                              </P>
+                            </FormRow>
+                          </Container>
+                        )}
+                        <Row>
+                          <Col>
+                            <Field
+                              name="phone"
+                              label={<Trans id="contactinfoPage.phoneNumber" />}
+                              component={Input}
+                              onBlur={handleBlur}
+                              onChange={handleChange}
+                              id="phone"
+                            />
+                          </Col>
+                          <Col>
+                            <Field
+                              name="extension"
```

Comment:
Attributes for this field are defined in ContactInfoFormSchema.js, why aren't they being used here?

Reply:
This field looks too big, do we want to reduce it's size?

Reply:
That is true, I will reduce the length of extension field